src/services/tracker-state.js

```diff
@@ -119,6 +119,23 @@ export default class TrackerState {
     return _.get(this.itemsForLocations, [generalLocation, detailedLocation]);
   }
 
+  getLocationForItem(itemName) {
+    const generalLocation = Object.keys(this.itemsForLocations).find(
+      (general) => Object.keys(this.itemsForLocations[general]).find(
+        (detailed) => this.itemsForLocations[general][detailed] === itemName,
+      ),
+    );
+    if (!generalLocation) {
+      return [null, null];
+    }
+
+    const detailedLocation = Object.keys(this.itemsForLocations[generalLocation]).find(
+      (detailed) => this.itemsForLocations[generalLocation][detailed] === itemName,
+    );
```

Comment:
We should be using [`_.find`](https://lodash.com/docs/#find) instead of the built in JavaScript methods in this situation.